features/sriov/deploy.sh

```diff
@@ -55,6 +55,14 @@ daemonset(){
   envsubst '\${nic} \${NODESELECTOR}' < ./10-sriov-daemonset.yaml | oc apply -f - > /dev/null || die "Error creating ds"
 }
 
+wait_for_ready(){
+  # This is to prevent creating the policy before the operator has been deployed like:
+  # error: unable to recognize “STDIN”: no matches for kind "SriovNetworkNodePolicy" in version "sriovnetwork.openshift.io/v1"
+  info "Waiting for the operator to be ready..."
+  # shellcheck disable=SC2154
+  while ! oc wait --for condition=ready pods -l name=sriov-network-operator -n "${operatornamespace}" --timeout=2400s; do sleep 10 ; done
```

Comment:
do we really need to wait 40 minutes for the operator to be ready

Reply:
It was a random choice. Any suggestions? 5 minutes would be ok?

Reply:
if it will be enough works for me:)

Reply:
I've set 300 by default but it can be tweaked if needed.